README.md

```diff
@@ -24,7 +24,7 @@ This includes a number of key Agent-based Modelling (ABM) building blocks such a
 ## Project Status
 <!-- Remove this section once it is no longer in pre-release / alpha state -->
 
-FLAME GPU 2 is currently in an pre-release (alpha) state, and although we hope there will not be significant changes to the API prior to a stable release there may be breaking changes as we fix issues, adjust the API and improve performance.
+FLAME GPU 2 is currently in an pre-release (alpha) state, and although we hope there will not be significant changes to the API prior to a stable release there may be breaking changes as we fix issues, adjust the API and improve performance. The use of *pure* Python agent functions (agent functiosn expressed as Python syntax which are transpiled to C++) is currently supported (see examples) but classes as an *experimental* feature.
```

Comment:
```suggestion
FLAME GPU 2 is currently in an pre-release (alpha) state, and although we hope there will not be significant changes to the API prior to a stable release there may be breaking changes as we fix issues, adjust the API and improve performance.
The use of *pure* Python agent functions (agent functions expressed as Python syntax which are transpiled to C++) is currently supported (see examples) but classed as an *experimental* feature.
```

Typo fixed above, no reason to not use a newline either. 

Not convinced by describing it as `currently supported` `but classed as an experimental feature`. Surely its just an experimental feature? Maybe:

```suggestion
FLAME GPU 2 is currently in an pre-release (alpha) state, and although we hope there will not be significant changes to the API prior to a stable release there may be breaking changes as we fix issues, adjust the API and improve performance.
The use of *pure* Python agent functions (agent functions expressed as Python syntax which are transpiled to C++) is currently experimental. See `examples/swig_purepython_boids/spatial3D` for an example.
```

I'm also not keen on it being called `pure python`. IMO we now have:

* CUDA C++
* CUDA C++ RTC
* Python 3 with CUDA C++ RTC agent functions
* Python 3 with (experimental, restricted subset of) Python 3 (and some c++ mixed in for good measure) agent functions)